test/e2e/storage/csi_volumes.go

```diff
@@ -78,13 +78,44 @@ var _ = Describe("PMEM Volumes", func() {
 	// List of testSuites to be executed in below loop
 	var csiTestSuites = []func() testsuites.TestSuite{
 		// TODO: investigate how useful these tests are and enable them.
+
 		// testsuites.InitMultiVolumeTestSuite,
+		// === above line refers to tests that are not found, proposing removal line from here
```

Comment:
Or update our copy of the E2E suite to 1.15, now that that is released. The test should be in that release.